### PR TITLE
Hannah & Seeb Home page

### PR DESCRIPTION
**Pull request by hannah68**
Please review our pull request

### CODE REVIEW

File: src/client/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom"
+import {FaFacebook, FaInstagram} from 'react-icons/fa';
+
+const Footer = () => {
+  return (
+		<footer className="footer-container">
+			<div className="footer">
+        <Link to="/" className="footer__logo">Pokimo & King</Link>
```
Comment (alexamaingard) [inline src/client/Footer.jsx:8]: The tabbing makes it a bit hard to read/understand what's going on inside the div. Not sure if it it GitHub's formatting or if it's like this in the actual file

**Review by alexamaingard — APPROVED** [review]

**Review by bravint — APPROVED** [review]

File: src/client/Footer.jsx (hunk 1 above)
Comment (hannah68) [reply src/client/Footer.jsx:8]: Thanks for the feedback. I fixed it.

Comment (glowkeeper) [reply src/client/Footer.jsx:8]: I agree with Alexa ;) Glad you fixed it, @hannah68 :)

**Comment by glowkeeper** [comment]
nice commit messages, guys :) Approved

**Pull request merged**